### DISCUSSION TITLE
Developer and Expermental No Fuel

### DISCUSSION THREAD

**Discussion by B1900Mech** [flybywiresim/aircraft#5333] [Help] [answered]
Hi,I did a clean install on both,I get no fuel on A/C with Simbrief flight plan,and unable to add any fuel thru Asobo fuel menu.

**Comment by delta5150** [comment]
I'm having the same issue. Did installs of stable, dev, experimental through installer. Did manual installs of all three. No joy. All the Asobo aircraft display correct fuel. It was working in the morning, then I updated from the installer and that's when the problem started.

**Reply by B1900Mech** [reply]
I changed from Lbs to Kg, or Kg to pounds,and reloaded A/C, that fixed this issue for me.

**Comment by tracernz** [comment, marked as answer]
Known issue with setting lbs through the FlyPad, workaround is to use the MCDU to set units. https://github.com/flybywiresim/a32nx/issues/5316#issuecomment-884086078
